_data/components.json

```diff
@@ -192,6 +192,43 @@
 		]
 	}
 }
+,{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Page success widget",
+		"fr": "Widget succ�s de page"
+	},
+	"description": {
+		"en": "Feedback widget for Canada.ca",
+		"fr": "Widget de r�troaction pour Canada.ca"
+	},
+	"modified": "2021-05-27",
+	"componentName": "gc-page-success",
```

Comment:
Gonna deal with this part manually since the links in GCWeb's index page derives folder names from that variable. I'll need to rename folders/files as a part of that change.

Reply:
Renamed this to ``gc-page-feedback`` (instead of ``gc-pg-hlpfl``) for consistency with the pages' file names.